KNR1050/iocBoot/iocKNR1050-IOC-02/config.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" ?>
+<ioc_config xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
Reference the config of IOC01 instead of duplicating it by replacing everything between the `<ioc_config>` tags with:
`<xi:include href="../iocKNR1050-IOC-01/config.xml"  />`